resources/template.html

```diff
@@ -140,22 +139,29 @@ <h2 class="mb-0">
 				</tr>
 			</table>
 			{{- else}}
+                        {{- $h := .Histogram }}
 			{{- with .Histogram}}
 			<table class="table table-striped table-bordered">
-				{{- range .Bucket}}
+                                {{- if le (len .Bucket) 0}}
```

Comment:
So there is the case of a "dual histogram", which has both classic buckets and the sparse buckets of a native histogram, so that the scraper can decide what kind of histogram it wants to ingest (possibly even both). In that case, we should probably display both the classic and the native histogram, maybe with a `<th scope="row">Native Histogram representation</th>`.

Reply:
I see, so would then make sense to format classic histograms if there are classic buckets and native histograms if `Schema != nil`?

Reply:
Schema is just an int32, so it never can be nil.

I would render a classic histogram if there are classic buckets, and I would render a native histogram if there are native buckets. The only edge case is if there are no buckets at all, but then the rendering of either isn't much of a difference (just sum and count), so you could pick any one rendering for that.